src/Microsoft.OData.Client/Serialization/ODataPropertyConverter.cs

```diff
@@ -687,6 +943,6 @@ private void AddTypeAnnotationNotDeclaredOnServer(string serverTypeName, ClientP
                 // and spatial values always contain their specific type inside the GeoJSON/GML representation.
                 primitiveValue.TypeAnnotation = new ODataTypeAnnotation(property.EdmProperty.Type.FullName());
             }
-        }
+        }     

```

Comment:
```suggestion
        }
```
Probably wasn't meant for checkin.

Reply:
Fixed